Automattic/wp-calypso
# WP.com Plugins: change Google Analytics item link to feature page

Issue: #4644

Change the link for google analytics plugin pointing to google analytics plan page

### testing

1) Go to site plugins page

2) Scroll down and click in google analytic plugin box

3) You should get the new google analytics page instead of WordPress support page
